tests/apps/migrations/auto/integration/test_migrations.py

```diff
@@ -632,65 +635,6 @@ def test_decimal_column(self):
             ),
         )
 
-    @engines_skip("cockroach")
-    def test_array_column_integer(self):
```

Comment:
Can we keep these tests somehow?

Reply:
Yes, we can. I don't remember why I rewrote those tests because now they work. I'll get those tests back.